zerver/views/message_summary.py

```diff
@@ -27,8 +27,8 @@ def get_messages_summary(
     if settings.TOPIC_SUMMARIZATION_MODEL is None:  # nocoverage
         raise JsonableError(_("AI features are not enabled on this server."))
 
-    if not (user_profile.is_moderator or user_profile.is_realm_admin):  # nocoverage
```

Comment:
I am assuming we do not need this check anymore, since we have added the setting.